mappings/net/minecraft/inventory/ClearableInventory.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS afy net/minecraft/inventory/ClearableInventory
```

Comment:
Inventory extends afy, not the other way around.

Reply:
I know, it's not an inventory, but rather an interface implemented by things that have an inventory that can be cleared. It has a single `clear()` method. `ClearableInventoryOwner` maybe?

Reply:
Uh. Does the clear() method *require* or *imply* that the cleared thing is an inventory?

Reply:
The clear method is called:
 - On block entities when using the /clone, /fill, /setblock commands and from a few blocks
 - Directly on inventories from containers and a few other places

Reply:
Yes I'd argue this is more like just _Clearable_ or _ClearableState_ or something

Reply:
Renamed it to `Clearable`.